### DISCUSSION TITLE
编译Alist

### DISCUSSION THREAD

**Discussion by Shame-xiaoen** [AlistGo/alist#7698] [Q&A]
看安安稳稳的教程编译完Alist 启动进web时无限转圈怎么解决

**Comment by anwen-anyi** [comment]
1. clone 前端没加 `--recurse-submodules` 参数
2. 语言包没初始化

**Comment by Shame-xiaoen** [comment]
大佬，有什么域名邮箱的项目吗



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2024年12月25日 21:43 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [AlistGo/alist] 编译Alist (Discussion #7698) |

clone 前端没加 --recurse-submodules 参数
语言包没初始化
